src/ifcopenshell-python/ifcopenshell/util/element.py

```diff
@@ -80,6 +80,15 @@ def get_properties(properties):
     for prop in properties or []:
         if prop.is_a("IfcPropertySingleValue"):
             results[prop.Name] = prop.NominalValue.wrappedValue if prop.NominalValue else None
+        elif prop.is_a("IfcPropertyEnumeratedValue"):
+            values = []
+            for enum in prop.EnumerationValues:
+                values.append(enum.wrappedValue)
+            if values:
+                results[prop.Name] = str(values) if len(values)>1 else values[0]
```

Comment:
Why not leave it as a list?

Reply:
Good question - I think I might have (falsely) expected an error from blender if the value wasnt strictly a string type.  I tested with your suggestion and it works fine :)